src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.wasm.cs

```diff
@@ -21,6 +23,53 @@ public partial class ScrollContentPresenter : ContentPresenter
 
 		private static readonly string[] HorizontalModeClasses = { "scrollmode-x-disabled", "scrollmode-x-enabled", "scrollmode-x-auto" };
 
+		public ScrollContentPresenter()
+		{
+			PointerReleased += ScrollViewer_PointerReleased;
+			PointerPressed += ScrollViewer_PointerPressed;
+			PointerCanceled += ScrollContentPresenter_PointerCanceled;
+		}
+
+		private void ScrollContentPresenter_PointerCanceled(object sender, Input.PointerRoutedEventArgs e)
+		{
+			HandlePointerEvent(e);
+		}
+
+		private void ScrollViewer_PointerPressed(object sender, Input.PointerRoutedEventArgs e)
+		{
+			HandlePointerEvent(e);
+		}
+
+		private void ScrollViewer_PointerReleased(object sender, Input.PointerRoutedEventArgs e)
```

Comment:
Does it make sense to intercept other pointer events this way? Or intercept at a lower level?

Reply:
I wondered. Actual scrollbars would, so maybe we should as well in this case.